components/Template.js

```diff
@@ -1,14 +1,29 @@
 var hogan = require('hogan.js');
 var React = require('react');
+var forEach = require('lodash/collection/forEach');
 
 class Template {
   render() {
     var content;
+    var data = this.props.data;
+    var templateHelpers = this.props.templateHelpers;
 
     if (typeof this.props.template === 'string') {
-      content = hogan.compile(this.props.template).render(this.props.data);
+      // We add all our template helper methods to the template as lambdas. Note
+      // that lambdas in Mustache are supposed to accept a second argument of
+      // `render` to get the rendered value, not the literal `{{value}}`. But
+      // this is currently broken (see
+      // https://github.com/twitter/hogan.js/issues/222).
+      forEach(templateHelpers, function(method, name) {
```

Comment:
**IF** you want to try it, you can use fat arrows everywhere here since you are using anonymous functions.